content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,175 @@
+==================
+Add new applicants
+==================
+
+Once an applicant submits an application, either using the online application, or emailing a job
+position alias, an applicant card is automatically created in the *Recruitment* application.
+
+However, in some instances, applicants may need to be created manually in the database. This could
+be necessary if, for example, a company accepts paper applications in the mail, or is meeting
+prospective applicants at an in-person job fair.
+
+To view current applicants, navigate to the :menuselection:`Recruitment` app, then click the desired
+job position card. Doing so reveals the :guilabel:`Applications` page, which displays all
+applicants for that specific role, in a default Kanban view, organized by stage.
+
+Add new applicants from a job position's :guilabel:`Applications` page by using either: the
+:ref:`New <recruitment/create-new-applicant>` button or the :ref:`quick add
+<recruitment/quick-add-applicant>` button.
+
```

Comment:
I liked how the **Quick add** section was before the **New applicant form** section, previously. 

I think this makes sense because in the instruction above you are already on the Kanban view of the Applications page, its a natural step to _quick add_ a applicant card, as well as the last instruction in this section is to optionally **Edit**– which takes you to the next, **New applicant form** section.

I also made a few edits in the copy here to streamline the instruction– lmk if you agree! 
```suggestion
.. _recruitment/quick-add-applicant:

Quick add
=========

On the :guilabel:`Applications` page, click the on the quick add button, represented by a small
:icon:`fa-plus` :guilabel:`(plus)` icon in the top-right corner of each stage to quickly add a new
applicant to that stage.

Enter the following information on the card:

- :guilabel:`Subject/Application`: enter the title for the card. Typically, this is the applicant's
  name, and job position being applied to. For example: `Laura Smith - HR Manager`. The text entered
  in this field is **not** visible in the Kanban view of the :guilabel:`Applications` page, unless
  the :guilabel:`Applicant's Name` field is left blank.
- :guilabel:`Applicant's Name`: enter the applicant's name. Displays as the card title in the
  Kanban view of the :guilabel:`Applications` page.
- :guilabel:`Email`: enter the applicant's email address.
- :guilabel:`Applied Job`: the current job position populates this field. If needed, the job
  position can be changed by selecting a different position from the drop-down menu. If a different
  job position is selected, after the card is created, the card appears on the
  :guilabel:`Applications` page for that newly-selected job position.

After the information is entered, click :guilabel:`Add`. The applicant appears in the list, and a
new blank applicant card appears.

If preferred, after entering the :guilabel:`Applicant's Name` in the Kanban card that appears, click
:guilabel:`Edit`, and a detailed applicant form loads. Refer to the :ref:`New applicant form
<recruitment/applicant-details>` section for details about filling out the form.

When doing a quick add, clicking away from an empty card, or clicking the :icon:`fa-trash-o`
(:guilabel:`trash`) icon, discards the applicant.

.. image:: add-new-applicants/quick-add.png
   :align: center
   :alt: All the fields for a new applicant form entered when using the Quick Add option.

```